docs/integrations/sources/simplecast.md

```diff
@@ -0,0 +1,32 @@
+# SimpleCast
+Say hello to the modern end-to-end podcasting platform. Simplecast remains the easiest way to get audio out to the world—with one-click publishing to Apple Podcasts Apple Podcasts , Spotify Spotify, or wherever your audience listens—and the best way for podcasters to cash in on their content.
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_token` | `string` | API Token. API token to use. Find it at your Private Apps page on the Simplecast dashboard. |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| podcasts | id | DefaultPaginator | ✅ |  ❌  |
+| episodes |  | DefaultPaginator | ✅ |  ❌  |
```

Comment:
@parthiv11 the contribution looks good, it misses a couple of primary key.
Example: 
* episodes = id
* timezone = value
* distribution_channels = id

Please take a look and see all stream and if there any other missing primary key.

Reply:
for `distribution_channels`  and `categories` stream PK is not possible

Reply:
Why not @parthiv11 ?

Reply:
they don't have unique values 

![image](https://github.com/user-attachments/assets/a2911b9f-23ed-40b2-aad3-094e103fb26e)

Reply:
Let's continue without PK, Thanks for sharing the info!